Add reCAPTCHA v3 guide (#175)
renuo/applications-setup-guide: ruby_on_rails/recaptcha.md

# reCAPTCHA v3

Here you will learn how to implement reCAPTCHA v3 into a project. It is pretty simple to use and doesn't hinder the users at all, as it all runs in the background.

## Implementation

Before you start, consider checking which page you would like to protect against suspicious activity (like spambots).

### Site registration

After you've decided, you can go to the [reCAPTCHA admin panel](www.google.com/recaptcha/admin) and register a new site. Make sure you're using the Renuo admin account.

To register the site, you can follow these steps:

> 1. Choose a label for the site, the project name should suffice.

Reviewer: ```suggestion
1. Use `[project-name]` as a label for the site.
```
